medusa/databases/main_db.py

```diff
@@ -699,7 +699,7 @@ def execute(self):
         utils.backup_database(self.connection.path, self.connection.version)
 
         log.info(u'Adding new quality field in the tv_episodes table')
-        self.connection.action('DROP TABLE IF EXISTS old_tv_episodes;')
+        self.connection.action('DROP TABLE IF EXISTS tmp_tv_episodes;')
```

Comment:
I don't think this should have been renamed.

Reply:
Trying to keep it consistant. Also tmp is a better description then old. As where not dealing with 'old' episodes.

Reply:
In that case you should also replace it in the next line.